### PR TITLE
Mention supervisor mode

### PR DESCRIPTION
**Pull request by vitvakatu**
Also use a link to PyPI instead of link to the repo readme (they are identical)

### CODE REVIEW

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -1404,7 +1409,7 @@ See [sample-config.yml][launcher-sample-config] for the final state of the
 configuration file.
 
 [exonum-launcher]: https://github.com/exonum/exonum-launcher
-[plugins-readme]: https://github.com/exonum/exonum-java-binding/blob/ejb/v0.9.0-rc2/exonum-java-binding/exonum_launcher_java_plugins/README.md
+[plugins-readme]: https://pypi.org/project/exonum-launcher-java-plugins/
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Does it have a compatibility table, if we use unversioned links (with plugins that is easier).

Comment (vitvakatu) [reply src/get-started/java-binding.md]: I've thought about it, we can use versioned link

File: src/get-started/java-binding.md [hunk 2]
```diff
@@ -1409,7 +1409,7 @@ See [sample-config.yml][launcher-sample-config] for the final state of the
 configuration file.
 
 [exonum-launcher]: https://github.com/exonum/exonum-launcher
-[plugins-readme]: https://pypi.org/project/exonum-launcher-java-plugins/
+[plugins-readme]: https://pypi.org/project/exonum-launcher-java-plugins/0.9.0rc1/
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: I'd immediately set it to 0.10.0 as it is not equal to the usual versions.

Comment (dmitry-timofeev) [reply src/get-started/java-binding.md]: Hence easy to miss the next time we update the versions

File: src/get-started/java-binding.md [hunk 3]
```diff
@@ -1117,9 +1117,14 @@ distributing the file.
 Provide a path to the resulting configuration file and a number of validator
 nodes in the network.
 
+Optional `supervisor-mode` parameter denotes the mode of the
+[Supervisor service][supervisor-service]. Possible values are "simple" and
```
Comment (bullet-tooth) [inline src/get-started/java-binding.md]:
```suggestion
[Supervisor service][supervisor-service]. Possible values are "simple" or
```

**Pull request merged**